transport/example/config_auth.json

```diff
@@ -0,0 +1,37 @@
+{
+    "auth_keys": {
+        "client": {
+            "type": "standard",
+            "key": "52abb3ac91971bb72bce17e7a289cd04476490b19e0d8eb7810dc42d4ac16c41"
```

Comment:
Noob question, config_auth.json and similar are not supposed to be committed to git I assume?